src/persistence/sql/cms_storage_sql_i.e

```diff
@@ -76,7 +76,7 @@ feature -- Operation
 				i := a_sql_statement.index_of (':', i)
 				if i = 0 then
 					i := n -- exit
-				else
+				elseif a_sql_statement.at (i-1).is_equal ('%'') or else  a_sql_statement.at (i-1).is_equal ('%"') or else a_sql_statement.at (i-1).is_equal (' ') or else a_sql_statement.at (i-1).is_equal ('=') then
```

Comment:
a_sql_statement [i - 1] = '%'' 
or else a_sql_statement [i - 1] = '%"' 
...

it is easier to read using bracket and '=' for character